apps/server/src/services/PlaybackService.js

```diff
@@ -127,6 +127,29 @@ export class PlaybackService {
     }
   }
 
+  /**
+   * Starts playback on next event
+   * @param {string} [fallbackAction] - 'stop', 'pause', or null
+   */
+  static startNext(fallbackAction) {
+    const nextEvent = eventLoader.findNext();
+    if (nextEvent) {
+      const success = PlaybackService.loadEvent(nextEvent);
+      if (success) {
+        socketProvider.info('PLAYBACK', `Loaded event with ID ${nextEvent.id}`);
+        PlaybackService.start();
```

Comment:
We also need to provide feedback for the play

Reply:
the play feedback comes from the start function

Reply:
Nice. Good catch